### DISCUSSION TITLE
Account flagged, support ticket not responded since 25 days - what can I do?

### DISCUSSION THREAD

**Discussion by rintisch** [community/community#53028] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
my account got blocked. I opened a support ticket the same day but got no answer after over 3 weeks. This is a problem for me because I cannot work properly for customers, e.g do Github Actions not run when they should be triggered (e.g. after a merge). 
Furthermore can customer not add me to new repositories, third party apps have no access to my repos, ...

Is there any option to get an answer to my support ticket? Is this long delay normal?

Thanks in advance!

**Comment by rintisch** [comment, marked as answer]
It just really took its time. No I got an answer and after a short conversation the account got unflagged.
It seems like you get a faster answer when choosing https://support.github.com/contact/reinstatement for your request.

**Comment by dvmakeworld** [comment]
Yeah, same for me, waiting like 2 weeks almost now and no answer yet.

**Comment by baharbaba** [comment]
I've been waiting for a response for 2 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by rintisch** [reply]
Have you tested https://support.github.com/contact/reinstatement ? Worked really fast for me.

**Reply by baharbaba** [reply]
Yes I saw this as your answer and submitted a ticket thank you for your response
